zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -47,8 +51,8 @@
     </names>
   </macro>
   <macro name="editor">
-    <names variable="editor translator" delimiter=", ">
-      <name delimiter="" prefix=" " and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
+    <names variable="editor translator" delimiter=". ">
```

Comment:
The way I see it the delimiter does not matter here because Zotero either writes 2 names separated by "and" or , if there are more than 2 names "name et al."